consensus/types/src/chain_spec.rs

```diff
@@ -780,8 +780,8 @@ impl YamlConfig {
             genesis_slot: chain_spec.genesis_slot,
             far_future_epoch: chain_spec.far_future_epoch,
             base_rewards_per_epoch: chain_spec.base_rewards_per_epoch,
-            deposit_chain_id: chain_spec.deposit_chain_id,
-            deposit_network_id: chain_spec.deposit_network_id,
+            deposit_chain_id: self.deposit_chain_id,
+            deposit_network_id: self.deposit_network_id,
```

Comment:
These should go in a different section above (their own section I think), rather than under the `Constants, not configurable` heading IMO

Reply:
Oh yep, missed that. Thanks.